src/tex/sources_table.tex

```diff
@@ -0,0 +1,12 @@
+\begin{tabular}{cc}
+\toprule
+Source & Count \\
+\midrule
+URL & 2288 \\
```

Comment:
First: very cool that you made this work!


I am wondering a bit about the URL thing. I almost feel that we should separate out the semi-automatically created questions, as otherwise, this high URL count might give the impression that we simply took almost everything from the web.

Reply:
Done the semi-automatically stuff